assets/styles/main.scss

```diff
@@ -28,3 +28,25 @@ $body-size: 20px;
   margin-bottom: 1rem;
 }
 
+//fix bottom padding of code blocks
+div.highlighter-rouge {
+  padding-bottom: 1em;
+}
+
+@media only screen and (max-width: 770px) {
```

Comment:
Why `770px`? Bulma doesn't use that number.

I think we should find a way that works with the framework, not against it.

Reply:
Tip: use the mixins that bulma provides. E.g.:

```scss
@include mobile {
  ...
}
```

Additionally it would make more sense to use flexbox, as bulma does, instead of floats. But maybe we're putting too much effort into a tiny detail that doesn't matter enough.

OTOH I think I would prefer to have a slightly too big menu that looks identical in most mobile browsers, over a different style in different browsers.

Reply:
flexbox sounds like too much work indeed.